### ISSUE TITLE
Can't import macros in index.html of themes

### ISSUE THREAD

**Issue by Keats** [getzola/zola#185]
Original: https://github.com/Keats/tera/issues/235

From a quick look the path do get rewritten correctly so it's not entirely clear what is happening

**Comment by Keats** [comment]
Ok found the issue, Tera only use the update paths in blocks in the renderer and use the AST `ImportMacro` node instead otherwise which is why it works in child pages but not the index one

**Comment by Keats** [comment]
@NuLL3rr0r 

https://github.com/Keats/tera/pull/218/commits/b87a920ab384b64f2bc78e4eb91a67fb582ec5d6 will fix it (tried locally). If you try the `next` branch, it should work

**Comment by NuLL3rr0r** [comment]
Sorry, but it seems the issue still persists for me. I even checked out the after-dark theme and it's the same. I built the next branch with head pointing to the following commit:

```
commit 6db79e77e84add9879fdc6808e08477f5ec58d11
Author: Vincent Prouillet <vincent@wearewizards.io>
Date:   Fri Nov 24 17:52:24 2017 +0100

    Update to Tera 0.11 beta
```

The same error:

```
Error: Failed to render page '/home/babaei/dev/babaei-net-blog-source/babaei.net/content/blog/2016-04-29-discourse-as-a-blog-comment-service-on-freebsd-without-docker.md'
Reason: Failed to render 'after-dark/templates/page.html' (error happened in 'after-dark/templates/index.html').
Reason: Template 'post_macros.html' not found
```

**Comment by Keats** [comment]
Can you do a cargo update to make sure it's using tera beta.1? I don't know
if I updated the cargo.lock.

**Comment by NuLL3rr0r** [comment]
That did it:

```
Updating registry `https://github.com/rust-lang/crates.io-index`
      Adding base64 v0.8.0
    Updating humansize v1.0.1 -> v1.0.2
    Updating plist v0.2.3 -> v0.2.4
    Updating tera v0.11.0-beta.0 -> v0.11.0-beta.1
    Updating xml-rs v0.6.1 -> v0.7.0
```

I rebuilt Gutenberg and it worked. Thank you!

**Comment by Keats** [comment]
Unrelated but if you have any opinions on https://github.com/Keats/gutenberg/pull/111 as you are writing in 2 languages, I'll be happy to hear them

**Comment by NuLL3rr0r** [comment]
Sure, I'll look into that and report back.

**Comment by Keats** [comment]
Fixed in 0.3.0 that is currently building in travis/appveyor
